src/styles/index.css

```
@@ -0,0 +1,9 @@
+/* empty source */
+
+@import "./variable.css";
+
+@import "./blocks/normalize.css";
+
+@import "./blocks/stars.css";
+
+@import "./blocks/card.css";
```

Don't put blank lines everywhere!
```suggestion
@import "./variable.css";

@import "./blocks/normalize.css";
@import "./blocks/stars.css";
@import "./blocks/card.css";
```